os/se/ameba/security_ameba_wrapper_tz.c

```diff
@@ -60,11 +60,19 @@
 #define FACTORYKEY_RTL_DEVICE (0x00010120)
 #define FACTORYKEY_RTL_CERT (0x00010122)
 
-/* SE Base, 1K for Cert, 1K for Key, 280K for SS, 4K reserve */
-#define SE_BASE CONFIG_AMEBAD_FLASH_CAPACITY - (292 * 1024)
+/* SE Base, After Bootloader before Kernel */
+#define SE_BASE 0xA000
```

Comment:
1. I think we should calculate the base address from partition table.
After writing or using that area, it should not be changed. But it does not mean every project should use the same address for that.
How about naming the part name as `ss` or something else?
And then get the offsets and calculate?
Could you refer below to get the definition of SE_BASE after calculating?
https://github.com/Samsung/TizenRT/blob/master/os/tools/mk_heap_regioninfo.sh
2. Please revert the file permission to 644.

Reply:
Is that necessary to calculate the address for Secure Storage, because this SE is platform dependent. Only rtl8721csm can use this SE.

Reply:
Let me ask you this first. When we request you to increase secure storage size, is this start address still fixed?

Reply:
Yes, the start address for Secure Storage is always fix, unless the reserve location for Bootloader is used up and need more space, then only start address of Secure Storage will be change.

Reply:
OK. Let keep it as you do.

Reply:
Could you check below?
```
How about naming the part name as ss or something else?
```
Noone knows what those partitions are. Then they can't know that they can remove it or they should keep them.

Reply:
Okay, I will rename them and add more specific comment.